### PR TITLE
Dev

### LINKED ISSUE #4: 02-container-registry
https://github.com/Arkobat/simple-fortune-cookie/blob/main/02-container-registry.md

### PR DESCRIPTION
**Pull request by Arkobat**
Closes #4

### CODE REVIEW

File: .github/workflows/simple-fortune-cookie.yaml [hunk 1]
```diff
@@ -16,3 +16,19 @@ jobs:
         run: |
           cd frontend
           go test
+
+  docker-push:
```
Comment (Arkobat) [inline .github/workflows/simple-fortune-cookie.yaml:20]: Should depend on test

File: .github/workflows/simple-fortune-cookie.yaml [hunk 2]
```diff
@@ -16,3 +16,19 @@ jobs:
...
+    name: Docker Push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v2
+      - name: Deploy to DockerHub
+        run: |
```
Comment (Arkobat) [inline .github/workflows/simple-fortune-cookie.yaml:32]: Needs to run `docker-compose create` before pushing

**Review by Arkobat — COMMENTED** [review]
Needs changes

**Review by Xcvbnm1515 — CHANGES_REQUESTED** [review]
Small mistakes

File: .github/workflows/simple-fortune-cookie.yaml [hunk 3]
```diff
@@ -20,15 +20,18 @@ jobs:
   docker-push:
     name: Docker Push
     runs-on: ubuntu-latest
+    depends_on: frontend-test
```
Comment (Arkobat) [inline .github/workflows/simple-fortune-cookie.yaml]: Needs to be a list

**Review by Arkobat — COMMENTED** [review]
Looks good

**Review by Xcvbnm1515 — APPROVED** [review]
Looks good

**Pull request merged**